### PR TITLE
[DEPENDENCY] Switch from "rimraf" to native "fs.rm"

### PR DESCRIPTION
**Pull request by matz3**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -173,6 +172,7 @@
 		"jsdoc": "^4.0.4",
 		"nyc": "^17.1.0",
 		"open-cli": "^8.0.0",
+		"rimraf": "^6.0.1",
```
Comment (flovogt) [inline package.json:175]: why you have added rimraf as a devdep?

Comment (codeworrior) [reply package.json:175]: Line 43?

Comment (flovogt) [reply package.json:175]: sorry, true. missed that

**Comment by matz3** [comment]
I've tested this on Windows via `ui5 build --all --clean-dest` and `ui5 build jsdoc --all --clean-dest` and didn't see any issues. So I'm eager to merge this.

Also see https://github.com/SAP/ui5-builder/pull/1098

**Review by codeworrior — APPROVED** [review]
LGTM

**Review by RandomByte — APPROVED** [review]

**Comment by matz3** [comment]
I forgot to adjust the commit message via squash merge. The merged commit on main has been adjusted as 1998257295be7038dd4222f1b241848c1fd0a05f.

**Pull request merged**